deepspeed/pt/deepspeed_light.py

```diff
@@ -119,8 +119,14 @@ def __init__(self,
         self.gradient_average = True
         self.warn_unscaled_loss = True
 
+        if dist_init_required is None:
+            dist_init_required = not dist.is_initialized()
+
+        self.dist_backend = "nccl"
         if dist_init_required:
-            dist.init_process_group(backend="nccl")
+            logging.info("Initializing torch distributed with backend: {}".format(
```

Comment:
Likewise, it would be good to add a `logging.warn("dist_init_required is a deprecated argument.")` or similar.

Reply:
Talked offline, waiting to deprecate for now.